src/api/accounts/businesses.md

```diff
@@ -23,20 +23,29 @@ A Business is associated with a single [Account][].
-| accountId     | String             | The Centrapay accountId for an org account.        |
-| accountName   | String             | The Centrapay account name for an org account.     |
-| test          | Boolean            | A flag which is present if the [Account][] is test |
-| nzbn          | String             | The unique NZBN identifier.                        |
-| name          | String             | Legal name recorded in the Companies Register.     |
-| tradingName   | String             | Trading name recorded in the Companies Register.   |
-| companyNumber | String             | Company number recorded in the Companies Register. |
-| createdAt     | {% dt Timestamp %} | When the Business was created.                     |
-| updatedAt     | {% dt Timestamp %} | When the Business was updated.                     |
-| createdBy     | {% dt CRN %}       | The User or API Key that created the Business.     |
-| updatedBy     | {% dt CRN %}       | The User or API Key that updated the Business.     |
+|     Field     |         Type          |                     Description                     |
+| :------------ | :-------------------- | :-------------------------------------------------- |
+| id            | String                | The unique identifier.                              |
+| accountId     | String                | The Centrapay accountId for an org account.         |
+| accountName   | String                | The Centrapay account name for an org account.      |
+| test          | Boolean               | A flag which is present if the [Account][] is test  |
+| nzbn          | String                | The unique NZBN identifier.                         |
+| name          | String                | Legal name recorded in the Companies Register.      |
+| tradingName   | String                | Trading name recorded in the Companies Register.    |
+| companyNumber | String                | Company number recorded in the Companies Register.  |
+| createdAt     | {% dt Timestamp %}    | When the Business was created.                      |
+| updatedAt     | {% dt Timestamp %}    | When the Business was updated.                      |
+| createdBy     | {% dt CRN %}          | The User or API Key that created the Business.      |
+| updatedBy     | {% dt CRN %}          | The User or API Key that updated the Business.      |
+| tax           | [Tax](#tax) {% opt %} | The value-added tax configuration for the Business. |
+
+### Tax
+
+|  Field  |  Type  |                              Description                              |
+| :------ | :----- | :-------------------------------------------------------------------- |
+| value   | String | Value of the tax.                                                     |
+| type    | String | Type of value-added tax. Can be `nz_gst`.                             |
+| country | String | Two-letter ISO code representing the country of the tax. Can be `NZ`. |
```

Comment:
I know we didn't originally plan to add `type` and `country`, but I don't see the downside of adding them here now. It saves us having to default these fields once we add other options in the future.

Reply:
We are moving ahead with `type`, but removing `country`. The `type` indicates what the `country` is anyways.